src/amp-story-player/amp-story-player-impl.js

```diff
@@ -211,6 +212,30 @@ export class AmpStoryPlayer {
     shadowRoot.appendChild(this.rootEl_);
   }
 
+  /**
+   * Helper to create a button.
+   * @private
+   */
+  initializeButton_() {
+    const option = this.element_.getAttribute('button');
+    if (!option || (option !== 'close' && option !== 'back')) {
+      return;
+    }
+    const optionBool = option === 'close';
+
+    const button = this.doc_.createElement('a');
+    button.classList.add('amp-story-player-' + option + '-button');
+    button.id = option + '-button';
+
+    button.textContent = optionBool ? '×' : '←';
```

Comment:
Let's add the icon in the CSS file if we can. We often prefer to use SVGs for icons as seen here:

https://github.com/ampproject/amphtml/blob/242e092ff440aa70334a4fdf9b06362c4a69ac56/extensions/amp-story/1.0/amp-story.css#L505

You should be able to find one in https://material.io/resources/icons